.travis.yml

```diff
@@ -89,9 +89,9 @@ jobs:
         - sudo apt-get update && sudo apt-get install -y python3-pip python3-setuptools && pip3 install --upgrade --user awscli
         - $(aws ecr get-login --no-include-email --region $AWS_REGION)
       script:
-        - cargo build --locked -p driver
-        - docker build --tag stablex-binary-public --build-arg SOLVER_BASE=gnosispm/dex-open-solver:$OPEN_SOLVER_VERSION --build-arg RUST_BASE=rust-binary -f docker/rust/Dockerfile .
-        - docker build --tag stablex-binary-private --build-arg SOLVER_BASE=163030813197.dkr.ecr.eu-central-1.amazonaws.com/dex-solver:$PRIVATE_SOLVER_VERSION --build-arg RUST_BASE=rust-binary -f docker/rust/Dockerfile .
+        - cargo build --locked -p driver --release
```

Comment:
Judging from your comment on the issue (that it would be sensible to keep running debug builds in staging), would it be easy enough to only build release on tagged builds and set the flag conditionally based on whether we are on a tag or master like this: https://travis-ci.community/t/how-do-i-conditionally-define-environment-variables/3376/2?

Reply:
I'd like to do this PR as is first and then we can change master to be release mode but with debug assertions and overflow checks.